plugins/airbrake/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@backstage/plugin-airbrake",
+  "version": "0.0.0",
```

Comment:
IMO the version should be start from 0.0.1 at least.

Reply:
Nah this is good, the first published version will be `0.1.0`. It's because we don't publish packages on merge, but on the first version bump

Reply:
Resolved: https://github.com/backstage/backstage/pull/8690#discussion_r776681377